### PR TITLE
Need to be added delete modal

### PR DESCRIPTION
**Pull request by StefanRudareanu**

### CODE REVIEW

File: src/Components/Admin/Admin.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from "react";
+import Header from "./Header/Header";
+import { Container, Modal, Grid, Box } from "@mui/material";
+import Cards from "./MettingRoom/Cards";
+import axios from "axios";
+import MeetingRoomForm from "./MettingRoomForm/MeetingRoom";
+import RoomApi from "../../HandleRequests/RoomApi";
+import { async } from "q";
```
Comment (daniel-mita) [inline src/Components/Admin/Admin.tsx:8]: Why import the npm package? Async is a reserved word in javascript and can be used without the need for external libraries.  Async - await is just syntactic sugar over the normal .then promise implementation. 
https://beyondthecloud.dev/blog/then-vs-async-await-in-lwc

File: src/HandleRequests/RoomApi.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { async } from "q";
```
Comment (daniel-mita) [inline src/HandleRequests/RoomApi.ts:2]: Same here for the async import

File: src/Components/Admin/Admin.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+const Admin = () => {
+    const [showmodal, setShowModal] = useState<boolean>(false);
+    const [datacontent, setDataContent] = useState<iCard[]>();
+    const [loaded, setLoaded] = useState(false);
+    const [showEditModal, setEditModal] = useState<boolean>(false);
+    const [editDataEvent, setEditData] = useState<iCard>();
+
+    const roomapi = RoomApi();
+    const handleSubmitForm = async (
+        Name: string | undefined,
```
Comment (alexgaita) [inline src/Components/Admin/Admin.tsx:23]: parameter names should start with lowercase

File: src/Components/Admin/Admin.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                    id={e.id}
+                    description={e.description}
+                    lastBooked={e.lastBooked}
+                ></Cards>
+            </Grid>
+        ));
+    };
+    useEffect(() => {
+        getDataContent();
+    }, []);
```
Comment (alexgaita) [inline src/Components/Admin/Admin.tsx:93]: Between these function there should be a new line
```suggestion
    }, []);
    
```

File: src/Components/Admin/Admin.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    useEffect(() => {
+        if (loaded != false) {
+            getDataContent();
+            setLoaded(false);
+        }
+    }, [loaded]);
```
Comment (alexgaita) [inline src/Components/Admin/Admin.tsx:99]: Same here, add a new line, or try to format using prettier

File: src/HandleRequests/RoomApi.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+function RoomApi() {
+    return {
+        async GetRooms() {
```
Comment (alexgaita) [inline src/HandleRequests/RoomApi.ts:4]: functions should start with lowercase

**Pull request merged**